cms/djangoapps/contentstore/tests/test_exams.py

```diff
@@ -0,0 +1,173 @@
+"""
+Test the exams service integration into Studio
+"""
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import ddt
+from django.conf import settings
+from edx_toggles.toggles.testutils import override_waffle_flag
+from pytz import UTC
+
+from cms.djangoapps.contentstore.signals.handlers import listen_for_course_publish
+from openedx.core.djangoapps.course_apps.toggles import EXAMS_IDA
+from xmodule.modulestore.tests.django_utils import TEST_DATA_MONGO_AMNESTY_MODULESTORE, ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
+
+@ddt.ddt
+@override_waffle_flag(EXAMS_IDA, active=True)
+@patch.dict('django.conf.settings.FEATURES', {'ENABLE_SPECIAL_EXAMS': True})
+@patch('cms.djangoapps.contentstore.exams._patch_course_exams')
+class TestExamService(ModuleStoreTestCase):
+    """
+    Test for syncing exams to the exam service
+    """
+    MODULESTORE = TEST_DATA_MONGO_AMNESTY_MODULESTORE
```

Comment:
@Zacharis278: I ran across this code while reviewing https://github.com/openedx/edx-platform/pull/31134

Please note that this is testing against Old Mongo, the Modulestore that's been deprecated for years and is currently in the process of being removed. Please use `TEST_DATA_SPLIT_MODULESTORE` (I believe it will default to this without the override line here).

Reply:
@ormsbee oh weird, yeah looks like I just blindly copied this from the older proctoring tests. Is cleaning this up blocking anything?

Reply:
@Zacharis278: That PR I linked to above is removing features from Old Mongo that this test depends on. In general, the team has tried to convert Old Mongo modulestore based tests to Split Modulestore based ones. Sometimes, a straightforward conversion fails (like this one, apparently). The policy we've adopted for tests that look useful but have never worked in Split Mongo is to mark them as `@skip("OldMongo Deprecation")` and review them later to see if they're worth keeping.

I realize this is new, actively developed functionality, and not some old test written 8+ years ago. I really hope it's a straightforward change for you folks. But we've been as loud as we could for a long while that Old Mongo is going away. So fixing this is non-blocking in the sense that we can land #31134 without it, but we would do that by disabling the test.

In terms of the timing, I don't think we'll merge before next week. It's a big PR that touches a bunch of old functionality, some of which I had forgotten even existed. I've been looking over for hours and I'm still less than halfway through it. Even assuming very quick turnaround on review cycles, I wouldn't expect the reviews to be done before Friday, and we wouldn't release something like this that close to the weekend.

Reply:
If there's proctoring code that relies on using Old Mongo, that should definitely get looked at as well. There was at least one that was a straightforward conversion (`cms/djangoapps/contentstore/tests/test_proctoring.py`), but I don't know if that was the one you were referring to.